src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: $main-font;
+  border: 1px solid $border-color;
```

Comment:
You use varibale "border-color" just once, so, perhaps you shouldn't make a new variable for the border color

Reply:
this mistake was fixed